engine/src/main/java/org/terasology/logic/players/LocalPlayer.java

```diff
@@ -114,7 +125,7 @@ public Vector3f getViewPosition() {
         return getViewPosition(new Vector3f());
     }
 
-    public Vector3f getViewPosition(Vector3f out) {
+    private Vector3f getViewPosition(Vector3f out) {
```

Comment:
Another access change API violation, same a little below. If not needed then lets not change it here :-)